### PR TITLE
Add digital literacy to the CS topics list for Curriculum Catalog

### PR DESCRIPTION
**Pull request by dmcavoy**
We noticed that the Jigsaw course did not have any topic or subject. After talking to the Curriculum team we decided a "Digital Literacy" topic would capture this tutorial and be useful for future tutorials especially in the K-2 space. So this PR adds "Digital Literacy" to the CS Topics list. 

I checked that it shows in LB on the Course Offering Edit Page

Screenshot 2023-05-02 at 7 26 35 AM

It shows in the Topics list on the Curriculum Catalog

Screenshot 2023-05-02 at 7 26 41 AM

And I added it on Jigsaw

Screenshot 2023-05-02 at 7 27 34 AM

### CODE REVIEW

File: apps/src/templates/teacherDashboard/CourseOfferingHelpers.jsx [hunk 1]
```diff
@@ -72,6 +73,7 @@ export const subjectsAndTopicsOrder = [
   'science',
   'english_language_arts',
   'history',
+  'digital_literacy',
```
Comment (megcrenshaw) [inline apps/src/templates/teacherDashboard/CourseOfferingHelpers.jsx:76]: Thanks for adding this here! I'm hoping the weirdly written tests caught this too 😅

File: apps/test/unit/templates/curriculumCatalog/curriculumCatalogCardTest.jsx [hunk 2]
```diff
@@ -84,9 +87,14 @@ describe('CurriculumCatalogCard', () => {
   it('renders one topic, the first available from ordered list, if no subject present', () => {
     render(<CurriculumCatalogCard {...defaultProps} topics={topics} />);
 
-    screen.getByText(subjectsAndTopicsOrder[firstTopicIndexUsed], {
-      exact: false,
-    });
+    screen.getByText(
+      translatedCourseOfferingCsTopics[
```
Comment (megcrenshaw) [inline apps/test/unit/templates/curriculumCatalog/curriculumCatalogCardTest.jsx:91]: Wow this is much cleaner! We shouldn't need `exact: false` now –– it was relying on the one-word keys, but this is a much better solution

**Review by megcrenshaw — APPROVED** [review]
Looks great! Would love for the `COURSE_OFFERING_CS_TOPICS` constant to coincide with the `translatedCourseOfferingCsTopics` but I believe @TurnerRiley added a test to cover that too?! Regardless, that's a separate work item.

File: dashboard/config/course_offerings/jigsaw.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "grade_levels": "K,1,2",
   "header": null,
   "image": "https://images.code.org/8fb543a9ed9f0fe3ed538e6d1bf5e189-Jigsaw2.png",
-  "cs_topic": null,
+  "cs_topic": "digital_literacy",
```
Comment (megcrenshaw) [inline dashboard/config/course_offerings/jigsaw.json:12]: I'm not super familiar with the best way to add this, but I'm under the impression that there's a 2-way sync between JSONs in levelbuilder and JSONs in prod?

File: apps/test/unit/templates/curriculumCatalog/curriculumCatalogCardTest.jsx (hunk 2 above)
Comment (dmcavoy) [reply apps/test/unit/templates/curriculumCatalog/curriculumCatalogCardTest.jsx:91]: Thanks to Turner for helping me figure this out!

File: dashboard/config/course_offerings/jigsaw.json (hunk 3 above)
Comment (dmcavoy) [reply dashboard/config/course_offerings/jigsaw.json:12]: This will get synced to levelbuilder during the DTL so it should be fine.

**Comment by dmcavoy** [comment]
> Looks great! Would love for the `COURSE_OFFERING_CS_TOPICS` constant to coincide with the `translatedCourseOfferingCsTopics` but I believe @TurnerRiley added a test to cover that too?! Regardless, that's a separate work item.

@megcrenshaw do you want to make a work item for this? I'm not sure exactly what you are thinking here.

**Pull request merged**